File: BaekJoon Online Judge/C++/Step08-Math_1 Mathematical Thinking/2869.cpp
//
//  2869.cpp
//  Step08-Math_1 Mathematical Thinking
//
//  Created by Yebin Kim on 2019/10/23.
//  Copyright © 2019 김예빈. All rights reserved.
//

#include <iostream>

using namespace std;

// Time Complexity O(1)
int main() {
    cin.tie(NULL);
    ios::sync_with_stdio(false);
    
    int a, b, v;
    int answer;
    
    cin >> a >> b >> v;
    
    answer = (a == v ? 1 : (v - a - 1) / (a - b) + 2);
    
    cout << answer;
    
    return 0;
}

File: BaekJoon Online Judge/Swift/Step08-Math_1 Mathematical Thinking/2869.swift
//
//  2869.swift
//  Step08-Math_1 Mathematical Thinking
//
//  Created by Yebin Kim on 2019/11/15.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

let input = readLine()!.split(separator: " ").map{ Int($0) }
var A = input[0]!
var B = input[1]!
var V = input[2]!

var answer = 1

if V != A {
    V -= A + 1
    answer = V / (A - B) + 2
}

print(answer)
